starter-code/index.html

```diff
@@ -0,0 +1,218 @@
+                          <p>
+                               probant an malis deserunt commodo labore labore do pariatur, culpa concursionibus singulis multos consequat. Nostrud anim eiusmod, ita aliqua fabulas.
+                          </p>
+                      </div>
+                  </div>
+                  <div class="col-md-4">
+                      <div class="text-center single-content">
+                          <i class="fa fa-flask"></i>
+                          <h3>Bootstrap</h3>
+                          <p>
+                               probant an malis deserunt commodo labore labore do pariatur, culpa concursionibus singulis multos consequat. Nostrud anim eiusmod, ita aliqua fabulas.
+                          </p>
+                      </div>
+                  </div>
+                  <div class="col-md-4">
+                      <div class="text-center single-content">
+                          <i class="fa fa-database"></i>
+                          <h3>Bootstrap</h3>
+                          <p>
+                               probant an malis deserunt commodo labore labore do pariatur, culpa concursionibus singulis multos consequat. Nostrud anim eiusmod, ita aliqua fabulas.
+                          </p>
+                      </div>
+                  </div>
+                  <div class="col-md-4">
+                      <div class="text-center single-content">
+                          <i class="fa fa-paint-brush"></i>
+                          <h3>Bootstrap</h3>
+                          <p>
+                               probant an malis deserunt commodo labore labore do pariatur, culpa concursionibus singulis multos consequat. Nostrud anim eiusmod, ita aliqua fabulas.
+                          </p>
+                      </div>
+                  </div>
+                  <div class="col-md-4">
+                      <div class="text-center single-content">
+                          <i class="fa fa-glass"></i>
+                          <h3>Bootstrap</h3>
+                          <p>
+                               probant an malis deserunt commodo labore labore do pariatur, culpa concursionibus singulis multos consequat. Nostrud anim eiusmod, ita aliqua fabulas.
+                          </p>
+                      </div>
+                  </div>
+                  <div class="col-md-4">
+                      <div class="text-center single-content">
+                          <i class="fa fa-wifi"></i>
+                          <h3>Bootstrap</h3>
+                          <p>
+                              probant an malis deserunt commodo labore labore do pariatur, culpa concursionibus singulis multos consequat. Nostrud anim eiusmod, ita aliqua fabulas.
+                          </p>
+                      </div>
+                  </div>
+              </div><!-- end row -->
+</div><!-- end container -->
+      </section>
+
+     
+           <section class="footer2">
```

Comment:
Instead of putting a class footer inside a section, you can directly write a < footer > tag :)